package.json

```diff
@@ -87,15 +88,13 @@
     "@babel/preset-env": "7.16.0",
     "@babel/register": "7.16.0",
     "babel-core": "7.0.0-bridge.0",
-    "babel-eslint": "10.1.0",
     "babel-gettext-extractor": "4.1.3",
     "babel-jest": "27.3.1",
     "babel-loader": "8.2.3",
     "comment-json": "4.1.1",
     "doctoc": "2.1.0",
-    "eslint-config-amo": "4.11.0",
+    "eslint-config-amo": "5.1.0",
     "eslint-plugin-async-await": "0.0.0",
```

Comment:
I am questioning the usefulness of this plugin: I don't think it is used in most of the other projects and we do use async/await everywhere now. I would like to remove this dev dependency: WDYT?

Reply:
@willdurand it looks that it does only provides two "code formatting" rules, which I would guess that prettier may also be taking care of. Also the package version is v0.0.0 and last release 5 years ago, definitely :+1:  on my side on getting rid of it.